CHANGELOG.md

```diff
@@ -1,3 +1,53 @@
+# v1.0.3 (2020-11-19)
+
+## OS Changes
+* Support setting Linux kernel parameters (sysctl) via settings ([#1158], [#1171])
```

Comment:
Should we provide an example of using this in a different section?

Reply:
Perhaps we should add this to the `README` or in the explanation of the settings.

Reply:
@tjk already added it to the readme https://github.com/bottlerocket-os/bottlerocket/blob/develop/README.md#kernel-settings

Reply:
Should I link to the README from the CHANGELOG? (Seems like a high risk of ending up with a broken link). Maybe just add (see README)?